tests/unit/mixins/test_mixin_methods.py

```diff
@@ -45,9 +45,47 @@ class M(GetMixin, FakeManager):
     assert isinstance(obj, FakeObject)
     assert obj.foo == "bar"
     assert obj.id == 42
+    assert obj._lazy is False
     assert responses.assert_call_count(url, 1) is True
 
 
+def test_get_mixin_lazy(gl):
+    class M(GetMixin, FakeManager):
+        pass
+
+    url = "http://localhost/api/v4/tests/42"
+
+    mgr = M(gl)
+    with responses.RequestsMock(assert_all_requests_are_fired=False) as rsps:
+        rsps.add(
+            method=responses.GET,
+            url=url,
+            json={"id": 42, "foo": "bar"},
+            status=200,
+            match=[responses.matchers.query_param_matcher({})],
+        )
+        obj = mgr.get(42, lazy=True)
+    assert isinstance(obj, FakeObject)
+    assert not hasattr(obj, "foo")
+    assert obj.id == 42
+    assert obj._lazy is True
```

Comment:
I get the explicitness but just want to say we're slowly moving away from pep8 (which our linters otherwise suggest) in our codebase, even for internal values that we already type-check.

> Don’t compare boolean values to True or False using ==:
> 
> ```
> # Correct:
> if greeting:
> ```
> 
> ```
> # Wrong:
> if greeting == True:
> ```
> 
> Worse:
> 
> ```
> # Wrong:
> if greeting is True:
> ```